kpatch/kpatch

```diff
@@ -163,7 +163,16 @@ load_module () {
 	fi
 
 	echo "loading patch module: $1"
-	insmod "$1" "$2"
+	for i in `seq 5`; do
+		if [[ $i != 1 ]]; then
+			echo "retrying..."
+			sleep 2
+		fi
+		insmod "$1" "$2" && break
+	done
+	[[ $i = 5 ]] && die "failed to load module $modname"
```

Comment:
If `insmod` happens to succeed on the 5th try (i = 5), then we will `die` here. Maybe set a fail flag within the loop (at i=5, if insmod fails, then set fail=1...) and then check it outside of the for loop?